packages/approval-signing-manager/src/lib/approval-signing-manager.ts

```diff
@@ -52,7 +52,9 @@ export class ApprovalSigner implements PolkadotSigner {
    * Get a signature from the approval key store. This may take a while as the approval process may require human approval
    */
   private async signData(address: string, data: Uint8Array): Promise<SignerResult> {
-    const message = `0x${Buffer.from(data).toString('hex')}`;
+    const fixedData = data.length > 256 ? blake2AsU8a(data) : data;
```

Comment:
Is there any unit test for this line?

Reply:
I added in tests to all signing managers.

As long as **this PR should be merged as two commits** and not squash and merge, the release shouldn't make unnecessary releases